index.html

```diff
@@ -24,21 +24,14 @@ <h4>
 
   <div class="news-column news-content">
     <ul>
+{%- assign items = site.news-items | reverse -%}
+{% for item in items offset: 1 limit: 3 %}
```

Comment:
Perhaps add a comment here or in the template file why offset 1 is used.

Alternatively, can't you just add the template to the `exclude` list in the config file instead? That seems cleanest and most explicit.

Reply:
That's a good idea, will try that.